src/psk.js

```diff
@@ -0,0 +1,40 @@
+const ILP = require('ilp')
+const BigNumber = require('bignumber.js')
+const compat = require('ilp-compat-plugin')
+const debug = require('debug')('superagent-ilp:psk')
+
+module.exports = async function handlePskRequest ({ res, payParams, maxPrice, plugin }) {
+  const [ destinationAmount, destinationAccount, sharedSecret ] = payParams
+  const { packet, condition } = ILP.PSK.createPacketAndCondition({
+    sharedSecret,
+    destinationAccount,
+    destinationAmount,
+    data: token
```

Comment:
I might be blind, but where is `token` defined?

Reply:
fixed